style.css

```diff
@@ -641,3 +641,477 @@ textarea::placeholder {
   border: none;
   background: #ff6b00;
 }
+
+@media only screen and (min-width: 768px) {
+  .union {
+    display: none;
+  }
+
+  .desktopitem {
+    flex: 0.8;
+    display: flex;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .item {
+    font-weight: 700;
+    font-size: 20px;
+    line-height: 20px;
+  }
+
+  .desktoptext {
+    color: #fff;
+    text-decoration: none;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+  }
+
+  #mail {
+    padding-top: 0.3%;
+  }
+
+  .headline {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    height: 100%;
+    align-items: center;
+    background-image: url("image/Illustration-header.png");
+    background-repeat: no-repeat;
+    background-position: right;
+    background-size: 38%;
+    padding: 0;
+  }
+
+  .headline:nth-child(2n) {
+    margin-right: 0;
+  }
+
+  .destopheading {
+    order: 2;
+    width: 40%;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    justify-content: center;
+    margin-left: 2.5%;
+  }
+
```

Comment:
- [ ]  I must say that you had a very beautiful design overall in the project which almost looks perfect 👏 .  The slight change needed has to do with the margin between your text introduction. Emphasis on this has to do with giving a visual appeal to your viewers. kindly fix it, and compliment the beautiful work you've done so far 👏 

# View:
![image](https://user-images.githubusercontent.com/50990925/214762409-7c500320-70a7-4aaa-85c9-b669abc85d4f.png)

# Figma:
![image](https://user-images.githubusercontent.com/50990925/214762436-5e90ee51-fbf2-432f-b39c-dddacfcae728.png)